### DISCUSSION TITLE
How to change default icons for custom ones ?

### DISCUSSION THREAD

**Discussion by arcadesfan** [spacedriveapp/spacedrive#1800] [General]
I like the file manager a lot, but would like to change the icons for MacOS Big Sur alike ones, or others, to match a Mac style better.

I would like to know if the default icons can be replaced for others, by editing the ones that are located in a folder. But I cannot find any folder containing the icon resources. Where are the default icons of spacedrive file manager located ? 

I will be grateful for your help.

**Comment by utkubakir** [comment]
We will have customizable themes in the future, it's not something we support at the moment.

**Comment by ketonik** [comment]
Thus far, i'm just thankful the folder are not YELLOW!!

I love the blue bordered translucent highlight too...nice touch, UI guys on this dev def got flava!

![image](https://github.com/spacedriveapp/spacedrive/assets/49221849/7876b124-3a11-4f1c-b0f8-80b4aebd1f22)
